src/module/item/license-sheet.ts

```diff
@@ -32,13 +33,22 @@ export class LancerLicenseSheet extends LancerItemSheet<EntryType.LICENSE> {
     for (let et of [EntryType.FRAME, EntryType.MECH_SYSTEM, EntryType.MECH_WEAPON, EntryType.WEAPON_MOD]) {
       let pack = game.packs.get(`world.${et}`);
       if (pack) {
-        let docs = (await pack.getDocuments({ system: { license: license.system.key } })) as any;
-        for (let d of docs as LancerItem[]) {
-          let rank = (d as any).system.license_level as number;
+        let index = await pack.getIndex();
+        // @ts-expect-error
+        let key = this.item.system.key;
+        for (let [id, index_data] of index.entries()) {
+          // @ts-expect-error
+          let item_license = index_data.system.license as string | undefined;
```

Comment:
I added a typedef for `LancerActor.system` and `LancerItem.system`, so these probably won't throw errors anymore.

Reply:
Never mind, it's not complaining about `LancerItem.system` here.

Reply:
Oh, thanks. I tried but just got an infinitely recursive type error, lol.